hive-llap/README.md

```diff
@@ -0,0 +1,143 @@
+
+# Hive LLAP Initialization Action
+This initialization action configures Google Cloud Dataproc to run Hive LLAP. For more information on Hive LLAP please visit: https://cwiki.apache.org/confluence/display/Hive/LLAP
+
+## Optional Configuration Options
+* num_llap_nodes: The number of nodes you wish LLAP to run on. This configuration will use 100% of the YARN resources LLAP is deployed on. Do not deploy on 100% of the deployed dataproc workers. You need available nodes in order to run the Tez Application Masters spawned by Hive sessions. Understand what level of concurrency you require to figure out how many workers you need available to support your users.
+* ssd: LLAP can extend the memory cache pool to local SSD on the workers it's deployed on. This increases the amount of data that can be cached to help with accelerating queries. This option may only be used when you include local ssd (num-worker-local-ssds flag) when deploying your dataproc cluster. Today we only support 1 SSD deployment. Note that if you use this option, you cannot start/stop your dataproc instance. 
+* exec_size_mb: This is the size of the executors running on LLAP. This is related to the hive.tez.container.size hive parameter. By default this number is 4096MB if not supplied directly. This knob allows you to configure clusters for uses cases where the default exec size may not be appropriate. 
+
+## Using this initialization action
+
+**:warning: NOTICE:** See [best practices](/README.md#how-initialization-actions-are-used) of using initialization actions in production.
+
+You can use this initialization action to create a new Dataproc cluster with LLAP enabled. When this initialization action runs, it will automatically download an additional script, llap_restart.sh. This script is used to auto-start LLAP and can be subsequently used to re-start LLAP if the cluster is stopped or shutdown. This script will reside on the the master node (master node 0 on HA deployments) on /usr/lib/hive_llap. If a user wants to download the initialization action from a seperate repo instead of the please specify the cloud storage bucket by adding the metadata parameter, init-actions-repo.
```

Comment:
Change `llap_restart.sh` to `start_llap.sh`. Change directory `/usr/lib/hive_llap` to `/usr/lib/hive-llap`. Mention that we look for the script in path `/hive-llap/start_llap.sh`, in case user specifies a separate repo.

Reply:
mention that the additional script will be downloaded from init actions bucket by default.

Reply:
done.